package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "tilejson-validator",
+  "version": "0.1.0",
+  "description": "Standards-based validation for TileJSON",
+  "author": "Azavea",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/azavea/tilejson-validator.git"
+  },
+  "scripts": {
+    "test": "eslint . && tap test/*.js --coverage"
+  },
+  "dependencies": {
+    "jsonlint-lines": "*"
```

Comment:
We've been burned in the past by `*` and `^X.X.X` dependencies. Consider specifying explicit dependencies versions, like how you have browserify specified.